modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js

```diff
@@ -431,6 +434,7 @@ ActionsDropdownRenderer.propTypes = {
 			onClick: PropTypes.string,
 			target: PropTypes.oneOf([
 				'async',
+				'event',
```

Comment:
Hello, @markocikos. Please, could you review these changes?

Reply:
@alinedoleron Please do not add new action `target`s. Our intent is to minimize and hopefully remove the whole API one day. This pattern is providing solutions tailored to individual customizations, which has no place in a general purpose component. If we continue doing this, we will end up with hundreds of targets.

Instead, please use `propsTransformer` API to add your custom action. See [docs](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2012217428/Frontend+Data+Set+Actions#Custom-Actions) and [a sample](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleFDSPropsTransformer.js).